proto/xudrpc.proto

```diff
@@ -67,7 +67,7 @@ service Xud {
     };
   }
   
-  /* Get a list of standing orders from the order book. */
+  /* Get a map between pair ids and its orders from the order book. */
```

Comment:
I'd either say "and their orders" or "and orders" since "its" is singular while "pair ids" is plural. It might also be more clear to say "buy and sell orders" since we are returning them in two separate collections.